configs/yoloxpai/yolox_pai_asff_tood_s_8x8_300e_coco.py

```diff
@@ -0,0 +1,4 @@
+_base_ = './yolox_pai_s_8x8_300e_coco.py'
+
+model = dict(bbox_head=dict(type='YOLOXTOODHead'))
```

Comment:
Doesn't this config use `neck=dict(type='YOLOXASFFPAFPN')`?

Reply:
I made a mistake. I fixed base config.